buildconfig/BuildConfig.cpp.in

```diff
@@ -12,6 +12,7 @@ Config::Config()
     LAUNCHER_DOMAIN = "@Launcher_Domain@";
     LAUNCHER_CONFIGFILE = "@Launcher_ConfigFile@";
     LAUNCHER_GIT = "@Launcher_Git@";
+    LAUNCHER_DESKTOPNAME = "@Launcher_DesktopName@";
```

Comment:
Name it `LAUNCHER_DESKTOPFILENAME` instead

Reply:
done